docs/_layouts/pages.html

```diff
@@ -70,3 +151,54 @@
     {{ content }}
   </div>
 </div>
+
+<script>
+
+  var filterInput = document.getElementById('filter-input');
+  var headings = document.getElementsByClassName('heading');
+  var tocElements = document.getElementsByClassName('example-names');
+  
+
+  filterInput.addEventListener('keyup', function (e) {
+    var value = this.value.toLowerCase();
+    var element;
+
+    for (i=0; i < headings.length; i++) {
+      if (!value || value == undefined || value == "" || value.length == 0) {
+        headings[i].classList.remove('display-none');
+      } else { 
+        headings[i].classList.add('display-none');
+      }
+    }
+
+
+ 
+    for (i=0; i < tocElements.length; i++) {
+      element = tocElements[i];
+    }
+
+    var match = function () {
+      return true;
+    };
+
+    var value = this.value.toLowerCase();
+
+    if (!value.match(/^\s*$/)) {
+      match = function (element) {
+        return element.innerHTML.toLowerCase().indexOf(value) !== -1;
+      };
+    }
+
+    for (i = 0; i < tocElements.length; i++) {
+      element = tocElements[i];
+      children = Array.from(element.getElementsByTagName('li'));
+      if (match(element) || children.some(match)) {
+        element.classList.remove('display-none');
+      } else {
+        element.classList.add('display-none');
+      }
+    }
+    
+
+  }); 
+</script>
```

Comment:
I don't see any changes to `site.js` and the search box has stopped working. Did you forget to commit the file?